sidecar.go

```diff
@@ -245,12 +247,16 @@ func (h *sidecarHandler) assembleConf(pod *corev1.Pod, className string) (telegr
 		if ok {
 			intervalConfig = fmt.Sprintf("interval = \"%s\"", intervalRaw)
 		}
+		versionConfig := ""
+		if versionRaw, ok := pod.Annotations[TelegrafMetricVersion]; ok {
+			versionConfig = fmt.Sprintf("metric_version = %s", versionRaw)
```

Comment:
One last nitpick. Could this be converted to `strconv.ParseInt` and throw an error if it fails?

I'm worried that someone accidentally sets the annotation incorrectly and things will break in a bad way.

Reply:
I updated to verify that the value is a number, but it does not validate that the value is either 1 or 2. The prometheus input in Telegraf makes an assumption based on the value so that validation was omitted.

Reply:
Thank you. I think it's fine as long as it's a number. Telegraf can throw an error if it's a wrong value.